pkg/image/qemu.go

```diff
@@ -136,7 +136,7 @@ func convertQuantityToQemuSize(size resource.Quantity) string {
 }
 
 func (o *qemuOperations) Resize(image string, size resource.Quantity) error {
-	_, err := qemuExecFunction(nil, nil, "qemu-img", "resize", "-f", "raw", image, convertQuantityToQemuSize(size))
+	_, err := qemuExecFunction(nil, nil, "qemu-img", "resize", "-f", "raw", "--preallocation=falloc", image, convertQuantityToQemuSize(size))
```

Comment:
Looks like we are missing "-o" here ?

Reply:
No, the resize doesn't take the -o, like the blank disk. https://bugzilla.redhat.com/show_bug.cgi?id=1414049#c19